### PR TITLE
Syllabus team minutes for 2022-02-02

### PR DESCRIPTION
**Pull request by alismith-dev**
## What does this change?

Module: N/A
Week(s): N/A

## Description

Minutes for syllabus team meeting on 2022-02-02.

## Who needs to know about this?

@CodeYourFuture/syllabus-team 

## Rendered Pages

-----
[View rendered docs/contributing/minutes.md](https://github.com/CodeYourFuture/syllabus/blob/minutes-2022-02-02/docs/contributing/minutes.md)

### CODE REVIEW

File: docs/contributing/minutes.md [hunk 1]
```diff
@@ -4,6 +4,67 @@ title: Team Minutes
...
+- Supporting schools with employment
+  - Cape Town have access to Barny
+  - Sally gave advice to keep working on projects etc
+  - Ali's direct experience: grads don't have lots of useful stuff on their Github
+    - [Graduate module](https://module-graduates.codeyourfuture.io/) is supposed to help with this
+      - But who's responsibility is this?
+      - Idea for improvement: after working on a group project, make sure they fork it onto their account, so that it shows up in the list of repos
+  - Some grads need further work before they're employable
+- WM update
+  - Have now broken the contract
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
  - Have now accepted the contract
```

File: docs/contributing/minutes.md [hunk 2]
```diff
@@ -4,6 +4,67 @@ title: Team Minutes
...
+  - Not following their "crazy terms"
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
  - 
```

GMAB! 😂

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]

File: docs/contributing/minutes.md (hunk 2 above)
Comment (alismith-dev) [reply docs/contributing/minutes.md]: LOL, I totally meant to remove this.... at least I didn't credit it 😂

File: docs/contributing/minutes.md (hunk 1 above)
Comment (alismith-dev) [reply docs/contributing/minutes.md]: Lol, I apparently totally misunderstood this

**Pull request merged**